src/js/components/tablero/Tablero.jsx

```diff
@@ -157,6 +202,7 @@ class Tablero extends Component {
       e.target.id = 'archive';
     }
     this.props.reorderIndicators({ oldIndex, newIndex }, e, type);
+    if (oldIndex !== newIndex) this.setState({ configModified: true });
```

Comment:
Because setState is asynchronous, the next line might not be properly evaluated. I suggest adding `isDragging` in this line and having the next line in else block.

Reply:
You are right ! I just made the change